source/codegen/metadata/niscope/functions.py

```diff
@@ -1882,7 +1803,7 @@
                     'mechanism': 'custom-code',
                     'value': '(num_samples * actual_num_wfms)'
                 },
-                'type': 'NIComplexNumber_struct[]'
```

Comment:
@reckenro I think these changes are also ones I'm not sure if need to be handled it seems the typedef struct for niscope has both names so it seems like it should be alright without the `_struct` see niScope.h for reference.

Reply:
Yeah, I think these changes are okay because it's the same struct just with different aliases in the niScope.h file like you said. So we'll see extra diffs in the generated files with these type name changes but they're the same structs so it's fine.

I do have some changes to better handle and provide overrides for custom_type param 'type's: https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/367950. I needed that for FGEN but I don't think it's required for your changes.